packages/lib/registry.ts

```diff
@@ -74,7 +81,7 @@ class Registry {
 		}
 	};
 
-	scheduleSync = async (delay: number = null, syncOptions: any = null) => {
+	scheduleSync = async (delay: number = null, syncOptions: any = null, overrideNetworkCheck: boolean = false) => {
```

Comment:
Could you explain the purpose of this extra parameter please? Is it not possible to get it working without it?

Reply:
It bypasses the network check. It's set to true when the function is called from the sidebar, when the user presses the sync button. I think it's the simplest way to still have manual sync.

Reply:
Ok I see what you mean. I feel "overrideNetworkCheck" is a little vague, please could you rename it to "doWifiConnectionCheck", which like now defaults to "false"?